### DISCUSSION TITLE
point_0745

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#422] [Roy]
"20190206T052011_20190206T052457_T43PFM"
![image](https://user-images.githubusercontent.com/54947195/147899959-e9131674-5916-4fd2-af18-7022468da14a.png)

@ryali93

**Comment by AntoniCesar** [comment]
20200129T051041_20200129T052502_T43PFM
![Captura](https://user-images.githubusercontent.com/86502862/154090578-4142faed-f3b1-4b97-b5b7-a2d5f8912354.JPG)

**Comment by AntoniCesar** [comment]
20190206T052011_20190206T052457_T43PFM
![Captura](https://user-images.githubusercontent.com/86502862/154096176-0dc2ae08-2395-42e4-a169-cc6f62f00885.JPG)
